internal/service/docdb/sweep.go

```diff
@@ -23,6 +23,212 @@ func init() {
+				DBClusterIdentifier: dBCluster.DBClusterIdentifier,
+			}
+
+			log.Printf("[INFO] Deleting DocDB Cluster: %s", id)
+
+			_, err := conn.DeleteDBCluster(input)
+
+			if err != nil {
+				log.Printf("[ERROR] Failed to delete DocDB Cluster (%s): %s", id, err)
+				continue
+			}
+
+			if err := WaitForDBClusterDeletion(context.TODO(), conn, id, DBClusterDeleteTimeout); err != nil {
+				log.Printf("[ERROR] Failure while waiting for DocDB Cluster (%s) to be deleted: %s", id, err)
+			}
+		}
+		return !lastPage
+	})
+
+	if sweep.SkipSweepError(err) {
+		log.Printf("[WARN] Skipping DocDB Cluster sweep for %s: %s", region, err)
+		return nil
+	}
+
+	if err != nil {
+		return fmt.Errorf("error retrieving DocDB Clusters: %w", err)
+	}
+
+	return nil
+}
+
+func sweepDBClusterSnapshots(region string) error {
+	client, err := sweep.SharedRegionalSweepClient(region)
+
+	if err != nil {
+		return fmt.Errorf("error getting client: %w", err)
+	}
+
+	conn := client.(*conns.AWSClient).DocDBConn
+	input := &docdb.DescribeDBClusterSnapshotsInput{}
+
+	err = conn.DescribeDBClusterSnapshotsPages(input, func(out *docdb.DescribeDBClusterSnapshotsOutput, lastPage bool) bool {
+		for _, dBClusterSnapshot := range out.DBClusterSnapshots {
+			name := aws.StringValue(dBClusterSnapshot.DBClusterSnapshotIdentifier)
+			input := &docdb.DeleteDBClusterSnapshotInput{
+				DBClusterSnapshotIdentifier: dBClusterSnapshot.DBClusterSnapshotIdentifier,
+			}
+
+			log.Printf("[INFO] Deleting DocDB Cluster Snapshot: %s", name)
+
+			_, err := conn.DeleteDBClusterSnapshot(input)
+
+			if err != nil {
+				log.Printf("[ERROR] Failed to delete DocDB Cluster Snapshot (%s): %s", name, err)
+				continue
+			}
+
+			if err := WaitForDBClusterSnapshotDeletion(context.TODO(), conn, name, DBClusterSnapshotDeleteTimeout); err != nil {
+				log.Printf("[ERROR] Failure while waiting for DocDB Cluster Snapshot (%s) to be deleted: %s", name, err)
+			}
+		}
+		return !lastPage
+	})
+
+	if sweep.SkipSweepError(err) {
+		log.Printf("[WARN] Skipping DocDB Cluster Snapshot sweep for %s: %s", region, err)
+		return nil
+	}
+
+	if err != nil {
+		return fmt.Errorf("error retrieving DocDB Cluster Snapshots: %w", err)
+	}
+
+	return nil
+}
+
+func sweepDBClusterParameterGroups(region string) error {
+	client, err := sweep.SharedRegionalSweepClient(region)
+
+	if err != nil {
+		return fmt.Errorf("error getting client: %w", err)
+	}
+
+	conn := client.(*conns.AWSClient).DocDBConn
+	input := &docdb.DescribeDBClusterParameterGroupsInput{}
+
+	err = conn.DescribeDBClusterParameterGroupsPages(input, func(out *docdb.DescribeDBClusterParameterGroupsOutput, lastPage bool) bool {
+		for _, dBClusterParameterGroup := range out.DBClusterParameterGroups {
+			name := aws.StringValue(dBClusterParameterGroup.DBClusterParameterGroupName)
+			input := &docdb.DeleteDBClusterParameterGroupInput{
+				DBClusterParameterGroupName: dBClusterParameterGroup.DBClusterParameterGroupName,
+			}
+
```

Comment:
We'll need to skip the default parameter groups. Something like

```suggestion
			if strings.HasPrefix(name, "default.") {
				log.Printf("[INFO] Skipping Document DB Parameter Group: %s", name)
				continue
			}
```

Reply:
Fixed :+1: